Overview.src.html

```diff
@@ -1315,8 +1315,20 @@ <h4 id="should-response-to-request-be-blocked-due-to-nosniff?"><dfn title="shoul
 <p>To perform a <dfn title=concept-fetch>fetch</dfn> using <var title>request</var>, run
 the steps below. An ongoing <span title=concept-fetch>fetch</span> can be
 <dfn title=concept-fetch-terminate>terminated</dfn> with reason <var title>reason</var>,
-which must be one of <i title>end-user abort</i>, <i title>fatal</i>, or
-<i title>timeout</i>.
+which must be one of <i title>end-user abort</i>, <i title>fatal</i>, <i title>timeout</i>, or
+<i title>garbage collection</i>.
+
+<p>The user agent may be asked to <dfn title=concept-fetch-suspend>suspend</dfn> the ongoing fetch.
+The user agent may either accept or ignore the suspension request. The susppended fetch can be
+<dfn title=concept-fetch-resume>resumed</dfn>. The user agent should ignore the suspension request
+if the ongoing fetch is updating the response in the HTTP cache for the request.
+
+<p class="note no-backref">The user agant does not update the entry in the HTTP cache for a
+<span title=concept-request>request</span> if request's cache mode is "no-store".
+
+<p class="note no-backref">The user agent does not update the entry in the HTTP cache for a
+<span title=concept-request>request</span> if `<code>Cache-Control: no-store</code>` HTTP
+header appears in the response. <span data-anolis-ref>HTTP</span>
```

Comment:
Can you combine these two notes into one? Seems a simple "or" would suffice to avoid a lot of redundant text.